src/pages/guides/nodejs/nitric-and-drizzle.mdx

```diff
@@ -0,0 +1,287 @@
+export const description = 'How to work with Nitric SQL Databases and Drizzle'
+
+# Nitric SQL Databases with Drizzle
+
+This guide demonstrates how to use [Drizzle ORM](https://orm.drizzle.team) with Nitric's [SQL database](/sql) feature. For this guide we will be creating a to-do app. The finished source can be found [here](https://github.com/nitrictech/examples/tree/main/v1/nitric-drizzle)
```

Comment:
Might be worth adding a note here that we support other ORMS aswell e.g. Prisma.

Reply:
There will be a prisma guide coming, will add it in that